utils/update_tools/update_tools.py

```diff
@@ -63,6 +63,15 @@ def solve_conflicts(g, branch=""):
     g.add(".")
 
 
+def merge_branches(location, branches):
+    """
+    Merge one or more branches into the current branch.
+    The branches have to be in string format
+    """
+
+    os.system("cd {} && git merge {} && cd -".format(location, branches))
```

Comment:
Why use os.system here, rather than gitpython?

Reply:
Looks like you need to set octopus=True?https://github.com/gitpython-developers/GitPython/issues/171

Reply:
I have tried multiple ways to do this, but it seems kinda complex to avoid the above described issue. `merge_base` should only find the common ancestor of the branches that are specified, without performing the merge. Probably `merge_tree` can be used, but I am no sure it works for Octopus merges.

IMO it would be better to leave it with the `os.system` call, at least we are sure that it works as we expect.

Reply:
Okay, rather than using `os.system`, please use `subprocess.check_call(xxx, shell=True)`

Reply:
Done